libbeat/publisher/client.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // Metrics that can retrieved through the expvar web interface.
 var (
-	publishedEvents = expvar.NewInt("libbeat.publisher.published_events")
+	publishedEvents = expvar.NewInt("publisher.events")
```

Comment:
Should we call it `publisher.events.count`? Otherwise we can't add any other values in the future to `publisher.events` as it would be a key and a namespace.